### DISCUSSION TITLE
discharge mode / sail mode

### DISCUSSION THREAD

**Discussion by pinkyellowed** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1286] [Q&A]
i have aldente pro, on the latest macbook pro and everything is up to par.
i have charge limit set at 60% and sail mode 20% lower than the charge limit with automatic discharge enabled.
am i understanding correctly, aldente will discharge automatically once charge limit is reached, recharging again once reached sail mode?
- my issue: once charge has reached my charge limit its been static at the 60% and i have to lower the charge limit just to have the discharging to begin, this undermines sail mode
- did i waste my money for sail mode, sail mode doesnt seem to be doing what its intended to do

**Comment by MatthiasKerbl** [comment]
Hi @pinkyellowed,

This is the intended behavior of the Sailing Mode.
It is a common misconception that the Sailing Mode does actively charge and discharge your MacBook between the sailing range. This would actually be much worse for the battery.
You can read a detailed description of what the Sailing Mode does on our blog "[Feature Explanation: Sailing Mode](https://apphousekitchen.com/feature-explanation-sailing-mode/)"

Best,
Matthias
